src/System.Management.Automation/FormatAndOutput/common/BaseFormattingCommand.cs

```diff
@@ -262,6 +262,9 @@ private void ProcessObject(PSObject so)
             }
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-linux`
Format-Custom with expression based EntrySelectedBy in a CustomControl.Property expression binding should be able to access the current object
```powershell
Expected strings to be the same, but they were different.
Expected length: 49
Actual length:   48
Strings differ at index 1.
Expected: '^^Entry se...'
But was:  '^Entry sel...'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Custom.Tests.ps1: line 410
410:         $ps.Invoke() -replace '\r?\n', "^" | Should -BeExactly $expectedOutput
```
Format-Custom with expression based EntrySelectedBy in a CustomControl.ScriptBlock expression binding should be able to access the current object
```powershell
Expected strings to be the same, but they were different.
Expected length: 62
Actual length:   61
Strings differ at index 1.
Expected: '^^Entry se...'
But was:  '^Entry sel...'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Custom.Tests.ps1: line 437
437:         $ps.Invoke() -replace '\r?\n', "^" | Should -BeExactly $expectedOutput
```